contracts/AdjudicationFramework.sol

```diff
@@ -2,25 +2,25 @@
 
 pragma solidity ^0.8.20;
 
-import "./mixin/BalanceHolder.sol";
```

Comment:
If you work on these contracts, and touching them. Then it would be nice to take them out of .solhintigonre file

Reply:
It's fiddly because parts of them are small variations on reality.eth contracts which have a different coding style, and changing the coding style to those parts to satisfy the linter will make them harder to compare to the previous (audited) versions. I'll try running the linter but disabling the parts I can't apply and see if that's still doing anything useful.

Reply:
OK, I enabled `solhint` but disabled a bunch of the rules inline in the contract. 
[9a083b3](https://github.com/RealityETH/subjectivocracy/pull/88/commits/9a083b37e10502460dce34073c235937198b8a1e)

The one I wasn't sure about was the length of the revert strings. I could make them shorter but it would make things less clear. I made this thread to see if someone knows the rationale behind this rule. 
https://old.reddit.com/r/ethdev/comments/17lxx5m/why_does_solhint_get_mad_about_long_revert_reasons/

Reply:
Bad-temperedly shortened the revert reason strings and enabled linting them
64ffb6c and 306b682